.woodpecker/.release.yml

```diff
@@ -0,0 +1,15 @@
+clone:
+  git:
+    image: plugins/git:next
+
+pipeline:
+  release:
+    image: golang:1.16
+    secrets: [github_token, docker_username, docker_password]
+    commands:
+      curl -sL https://git.io/goreleaser | bash
```

Comment:
Wouldn't having a pinned version make more sense?

Reply:
@inverse You're right. Just copied that code from the official docs from goreleaser, but version pinning should be preferred. What do you think about using the goreleaser docker image directly instead of a golang image?